test/scenarios-bootc/releases/el96-lrel@fips.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+start_image="rhel96-bootc-brew-${LATEST_RELEASE_TYPE}-with-optional-fips"
+
+check_platform() {
+    if [[ "${UNAME_M}" =~ aarch64 ]] ; then
+        record_junit "setup" "scenario_create_vms" "SKIPPED"
+        exit 0
+    fi
+}
+
+scenario_create_vms() {
+    if ! does_commit_exist "${start_image}"; then
```

Comment:
for scenario-bootc the function must be `does_image_exist()`

```suggestion
    if ! does_image_exist "${start_image}"; then
```